test/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge, chrome=1">
+    <meta charset="utf-8">
+    <title>React Dart Tests</title>
+</head>
+<body>
+    <ul>
+        <li>
+            <a href="/child_key_warning_test.html">
+                Run <code>child_key_warning_test.html</code>
+            </a>
+        </li>
+        <li>
+            <a href="/lifecycle_test.html">
+                Run <code>lifecycle_test.html</code>
+            </a>
+        </li>
+        <li>
+            <a href="/react_test_utils_test.html">
+                Run <code>react_test_utils_test.html</code>
+            </a>
+        </li>
```

Comment:
Niiiiiiiiice. Would you mind adding links to the `js_interop_helpers_test` files as well, while you're in here?